### DISCUSSION TITLE
Drum tuning (and a drumkit editor)

### DISCUSSION THREAD

**Discussion by github-wlk** [hydrogen-music/hydrogen#1136] [Ideas]
Hi, I use hydrogen on Linux, and here is an idea that I would like to see.

Tuning. Mass tuning, perhaps, to be more precise. What I'm imagining... A drumkit is selected. You pull up a window which will list all the instruments in that drumkit, along with the tuning. A dial or field to modify all the tuning for that drumkit. Checkboxes next to each instrument to determine whether you want that tuning change to apply (you want to change the pitch of the toms, but not the cymbals, etc). You see what I'm going at.... A master drum-tuning page. Then, you can save that drumkit as a variant (So you'd have the original drumkit X, and now also, say, "X in Bb"... Same set of wavs) (for "variant", see https://github.com/hydrogen-music/hydrogen/discussions/1134)

Master drumkit editor would also allow mass changes to midi channel, etc.

Thanks!

**Comment by theGreatWhiteShark** [comment]
Hey @github-wlk,

In the latest version on github (you have to compile the master branch) we already have this feature (was introduced in #1000).

Every instrument has now a main pitch knob to alter the tuning of all its layers at once.
